framer/Utils.coffee

```diff
@@ -497,16 +504,7 @@ Utils.modulate = (value, rangeA, rangeB, limit=false) ->
 # STRING FUNCTIONS
 
 Utils.parseFunction = (str) ->
-
-	result = {name: "", args: []}
-
-	if _.endsWith str, ")"
-		result.name = str.split("(")[0]
-		result.args = str.split("(")[1].split(",").map (a) -> _.trim(_.trimEnd(a, ")"))
-	else
-		result.name = str
-
-	return result
+	Animator.parseFunction(str)
```

Comment:
But also not move animatorFor then?
The problem is that I want to use `animatorFor` in my code panel code, but Animator is not public in Framer. So the choices are putting it in Utils or make Animator public. Would you go for the latter instead?